scripts/generate-inline-script-hash.js

```diff
@@ -0,0 +1,29 @@
+var sjcl = require("../src/assets/scripts/sjcl.js");
+var fs = require("fs");
+
+fs.readFile("./www/index.html", "utf8", (error, data) => {
+  if (!error) {
+    let validResults = [];
+    let result = data.match(/<script[\s\S]*?>[\s\S]*?<\/script>/gi);
+    for (let i = 0; i < result.length; i++) {
+      let resultItem = result[i];
+      if (!resultItem.includes("><")) {
+        let innerContent = resultItem.split(">")[1].split("<")[0];
```

Comment:
Won't this break if the inner content of the script contains a less-than or less-than-or-equal to comparison?

e.g.

```
<script>
  var i = 0;
  if(i < 10)
    console.log("less than 10")
</script>
```

Reply:
Sweet, nice catch. Changed < to </